roles/network_plugin/calico/tasks/install.yml

```diff
@@ -181,10 +181,31 @@
           "logSeverityScreen": "Info",
           "nodeToNodeMeshEnabled": {{ nodeToNodeMeshEnabled|default('true') }} ,
           "serviceExternalIPs": {{ _service_external_ips|default([]) }},
+          "serviceClusterIPs": [{"cidr": {{ kube_service_addresses }} }],
```

Comment:
I think CALICO_ADVERTISE_CLUSTER_IPS env var should be removed from the calico-node as well

Reply:
Removed it.  It is entirely unused.